engine/apps/api/serializers/organization.py

```diff
@@ -113,14 +113,23 @@ class Meta:
 class CurrentOrganizationConfigChecksSerializer(serializers.ModelSerializer):
     is_chatops_connected = serializers.SerializerMethodField()
     is_integration_chatops_connected = serializers.SerializerMethodField()
+    mattermost = serializers.SerializerMethodField()
```

Comment:
I wouldn't change this API for this, since this is being used outside oncall to confirm a minimal setup required steps. If we need mattermost specific info, maybe we need a new endpoint?

Reply:
I was planning on using the existing 
API that we have used for telegram found this note in that flow hence moved the logic 
over here. https://github.com/grafana/oncall/blob/dev/engine/apps/api/serializers/organization.py#L91

Reply:
Hm... I think the comment refers to something else (the comment was added [here](https://github.com/grafana/oncall/pull/1713/files#diff-5e7f252d928ee561501ecdf2c8f730cc0259c58fc49eb4d5e5880b41b4ee2a4bR117)). The API here was added much later and it is used for a 3rd party integration, so it would be better if we can leave that one unchanged (and add a different endpoint or for now just update the one you linked despite the comment there).